packages/client/hmi-client/src/components/workflow/tera-simulate.vue

```diff
@@ -0,0 +1,201 @@
+				label="Add Chart"
+				icon="pi pi-plus"
+			></Button>
+		</div>
+		<div v-else-if="activeTab === SimulateTabs.input && node" class="simulate-container">
+			<div class="simulate-model">
+				<Accordion :multiple="true" :active-index="[0, 1, 2]">
+					<AccordionTab>
+						<template #header> {{ node.inputs[0].value?.[0].model.name }} </template>
+						<model-diagram :model="node.inputs[0].value?.[0].model" :is-editable="false" />
+					</AccordionTab>
+					<AccordionTab>
+						<template #header> Model configurations ({{ simConfigs.length }}) </template>
+						<DataTable
+							v-if="node.inputs[0].value?.length"
+							class="model-configuration"
+							showGridlines
+							:value="simConfigs"
+						>
+							<ColumnGroup type="header">
+								<Row>
+									<Column selection-mode="multiple" headerStyle="width: 3rem" />
+									<Column v-for="(v, i) of simVars" :key="i" :header="v" />
+								</Row>
+							</ColumnGroup>
+							<Column selection-mode="multiple" headerStyle="width: 3rem" />
+							<Column v-for="(value, i) of simVars" :key="i" :field="value"> </Column>
+						</DataTable>
+					</AccordionTab>
+					<AccordionTab>
+						<template #header> Simulation Time Range </template>
+						<div class="sim-tspan-container">
+							<div class="sim-tspan-group">
+								<label for="1">Units</label>
+								<Dropdown
+									id="1"
+									class="p-inputtext-sm"
+									v-model="openedWorkflowNodeStore.tspanUnit"
+									:options="TspanUnitList"
+								/>
+							</div>
+							<div class="sim-tspan-group">
+								<label for="2">Start date</label>
+								<InputNumber
+									id="2"
+									class="p-inputtext-sm"
+									v-model="openedWorkflowNodeStore.tspan[0]"
+									inputId="integeronly"
+								/>
+							</div>
+							<div class="sim-tspan-group">
+								<label for="3">End date</label>
+								<InputNumber
+									id="3"
+									class="p-inputtext-sm"
+									v-model="openedWorkflowNodeStore.tspan[1]"
+									inputId="integeronly"
+								/>
+							</div>
+						</div>
+					</AccordionTab>
+				</Accordion>
+			</div>
+		</div>
+	</section>
+</template>
+
+<script setup lang="ts">
+import { computed, ref } from 'vue';
+import Accordion from 'primevue/accordion';
+import AccordionTab from 'primevue/accordiontab';
+import Column from 'primevue/column';
+import Row from 'primevue/row';
+import ColumnGroup from 'primevue/columngroup';
+import DataTable from 'primevue/datatable';
+import Dropdown from 'primevue/dropdown';
+import Button from 'primevue/button';
+import InputNumber from 'primevue/inputnumber';
+
+import { useOpenedWorkflowNodeStore } from '@/stores/opened-workflow-node';
+import { TspanUnits } from '@/types/SimulateConfig';
+
+import SimulateChart from './tera-simulate-chart.vue';
+import ModelDiagram from '../models/tera-model-diagram.vue';
+
+const openedWorkflowNodeStore = useOpenedWorkflowNodeStore();
+
+enum SimulateTabs {
+	input,
```

Comment:
Nitpick: can we follow current convention for enums which seems to just be capital case

Reply:
i tried that, i get this, if thats the convention i dont know how the same error isnt being thrown
<img width="1084" alt="Screenshot 2023-06-02 at 11 26 18 AM" src="https://github.com/DARPA-ASKEM/Terarium/assets/14062458/fb548ee1-44bb-4a8f-ae3a-06668fc925e9">

Reply:
Huh I could have sworn I saw caps case enums somewhere. Well there you go! Thanks shawn.